rundeckapp/grails-spa/packages/ui-trellis/src/app/utilities/tests/nodeUi.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { glyphiconBadges } from "../nodeUi";
+
+describe("glyphiconBadges", () => {
+  it("parses well-formed input string correctly", () => {
+    const input = "glyphicon-user1, fa-badge2, fab-badge1, glyphicon-lock-1";
+    expect(glyphiconBadges({ "ui:badges": input })).toEqual([
+      "glyphicon-user1",
+      "fa-badge2",
+      "fab-badge1",
+      "glyphicon-lock-1",
+    ]);
+  });
```

Comment:
Please add a value in the input that does not end with a number, ie `fa-house-user glyphicon-download-alt`

they are both actual icons, and the glyphiconBadges should return them as well.